web/components/feed/activity-items.ts

```diff
@@ -520,6 +520,13 @@ export function getRecentContractActivityItems(
   return [questionItem, ...items]
 }
 
+function commentIsGeneralComment(comment: Comment, contract: Contract) {
+  return (
+    comment.answerOutcome === undefined &&
+    (contract.outcomeType !== 'BINARY' ? comment.betId === undefined : true)
```

Comment:
Stephen is adding numerical markets soon. Might be better to check if it is equal to 'FREE_RESPONSE' instead.